docs/builders/joystick-customization.md

```diff
@@ -0,0 +1,305 @@
+
+2. The index of which axis on the controller you want to bind the camera movement to. This is distinct for the type of controller you will use and to find these values for a new controller see [Setup new joystick type](#Setup-new-joystick-type). If you are using an already supported controller, you can use the "map" at the top of the asset to find the indices. Either you can put in the indices directly or you can use the map with the descriptive name such as `controller.RightTrigger` or `controller.LeftThumbStick.LeftRight`.
+
+3. The type of camera movement you want this axis to do. This defines how the camera will move in OpenSpace when you move the specified axis of the controller. Must be one of the identifiers in the following list:
+
+| Identifier   | Description                                                             |
+|--------------|-------------------------------------------------------------------------|
+| "None"       | Unbinds the axis, no camera movement is applied. Default value from the start. |
+| "Orbit X"    | Move the camera in the left/right direction in relation to the focus, while still keeping the same distance. The camera will move as if it was orbiting the focus. |
+| "Orbit Y"    | Move the camera in the up/down direction in relation to the focus, while still keeping the same distance. The camera will move as if it was orbiting the focus. |
+| "Zoom"       | Move the camera closer or further away from the focus                   |
+| "Zoom In"    | Move the camera closer to the focus                                     |
+| "Zoom Out"   | Move the camera further away from the focus                             |
+| "LocalRoll"  | Roll the camera (clockwise) in relation to the middle of the view       |
+| "GlobalRoll" | Roll the camera (clockwise) in relation to the focus                    |
+| "Pan X"      | Turn the camera left/right in relation to the view                      |
+| "Pan Y"      | Turn the camera up/down in relation to the view                         |
+
+4. (Optional) Whether or not this axis should be inverted. This is a common setting in video games. Defaults to `false`.
+
+5. (Optional) The type of joystick that this axis represents on the controller. The options are either `"JoystickLike"` or `"TriggerLike"`. A joystick is `"TriggerLike"` if it can only be pressed or pushed in one direction. A `"JoystickLike"` axis can be pushed in two directions, for example, left **and** right, or up **and** down. Defaults to `"JoystickLike"`.
+
+6. (Optional) Whether or not this axis is "sticky". In most cases, this should be set to `false`. An axis is "sticky" if, when you let go of it, the values it represents in the software do not go back to the default. Another sign is that the longer you push it, the value, and therefore the movement, gets more and more extreme over time and does not stop or slow down when you let go. Defaults to `false`.
```

Comment:
Which one are you referring to? Argument 5 or 6? I'm not sure a throttle is a good example for number 6 but it could be for number 5 instead, although there I think the trigger is an easier example.

Number 6 is really hard to explain in a brief way because it is all about the values that are sent and has nothing to do with how the actual joystick looks like. It can look like it goes back to the "origin" when it is let go, but the values it sends don't go back, they just stay where they were. And when you press the joysticks longer over time the values just keep going even beyond the -1 to 1 range, it could go on to really high values. So the value we use instead then is the difference between the new and the old value instead of the value itself. We sort of re-normalize it every frame instead of taking the raw value.

Reply:
Ah.. yea, I meant option 6..   so in that case let's leave it as it is